docs/developing-fuzzbench/adding_a_new_benchmark.md

```diff
@@ -161,18 +212,80 @@ export FUZZER_NAME=afl
 export BENCHMARK_NAME=libpng-1.2.56
 
 make build-$FUZZER_NAME-$BENCHMARK_NAME
+
+# This command will fuzz forever. Press Ctrl-C to stop it.
 make run-$FUZZER_NAME-$BENCHMARK_NAME
 ```
 Finally, add your benchmark to the list of OSS-Fuzz benchmarks in
 [test_fuzzer_benchmarks.py](https://github.com/google/fuzzbench/blob/master/.github/workflows/test_fuzzer_benchmarks.py)
 
 This ensures that CI tests your benchmark with all fuzzers.
 
-## Testing the benchmark in CI
+## Submitting the benchmark in a pull request
 
-Add your benchmark to the `STANDARD_BENCHMARKS` list in
+Add your benchmark to a list in
 [build_and_test_run_fuzzer_benchmarks.py](https://github.com/google/fuzzbench/blob/master/.github/workflows/build_and_test_run_fuzzer_benchmarks.py)
-so that it will be tested in CI.
+so that it will be tested in CI. If your benchmark is a standard benchmark, add
+it to the `STANDARD_BENCHMARKS` list, otherwise add it to the the
+`OSS_FUZZ_BENCHMARKS` list.
 
 If everything works, submit the integration in a
 [GitHub pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+
+## Overview of how builds work in FuzzBench
+
+We don't think most end users (people who integrate benchmarks or fuzzers into
```

Comment:
For "people who integrate benchmarks" it's probably useful to know this though, no?

Reply:
I think it's not that important for them to know. But I don't have a strong opinion on whether I need to qualify this section like this. Let me know if you think I should remove it.

Reply:
i'd probably drop the part in the parenthesis, but no strong opinion, up to you

Reply:
Done.